frontend/src/libs/components/author/author.tsx

```diff
@@ -0,0 +1,48 @@
+import { type FC } from 'react';
+
+import { Avatar } from '../avatar/avatar.jsx';
+import styles from './styles.module.scss';
+
+type Properties = {
+  name?: string;
+  followers?: number;
+  rating?: number;
+  date?: string;
+  readingTime?: string;
+  genre?: string;
+};
+
+const Author: FC<Properties> = ({
+  name = 'Todd Demoer',
+  followers = 10,
+  rating = 700,
+  date = 'May 28',
+  readingTime = '7 min',
+  genre = 'Fiction',
+}) => {
```

Comment:
```suggestion
}) => (
    <>
      <div className={styles.container}>
        <div className={styles.avatarWrapper}>
         <Avatar username={name} avatarUrl={null} />
        </div>
        <div>
          <h1>{name}</h1>
          <div>
            <span>{followers}following</span>
            <span>{rating}rating</span>
          </div>
        </div>
      </div>
      <div>
        <div>
          <span>{date}</span>
          <span>{readingTime}</span>
        </div>
        <h2>{genre}</h2>
      </div>
    </>
  );
```

Reply:
Oo. Sorry this is old folder from first merge with wr-53. I thougth that it will be deleted after merging with updated wr-53 branch (author-component). But it seems like it is stayed in my branch somehow.  I already deleted this useless common author-component and it seems like in the pages/article/components/author.tsx everything fine.